go/vt/wrangler/materializer.go

```diff
@@ -64,7 +64,7 @@ const (
 
 // MoveTables initiates moving table(s) over to another keyspace
 func (wr *Wrangler) MoveTables(ctx context.Context, workflow, sourceKeyspace, targetKeyspace, tableSpecs,
-	cell, tabletTypes string, allTables bool, excludeTables string) error {
+	cell, tabletTypes string, allTables bool, excludeTables string, autoRun, stopAfterCopy bool) error {
```

Comment:
Is it intentional to call the function argument `autoRun` instead of `autoStart`? I would have kept it consistent with the flag, but maybe there's a good reason to name it differently..

Reply:
Oopsie: had started with `auto_run` and changed it to `auto_start` because I thought it would be more consistent with with the Start command and missed modifying these.